src/main/java/uk/gov/hmcts/reform/hmc/controllers/HearingManagementController.java

```diff
@@ -37,13 +37,17 @@
 @Validated
 public class HearingManagementController {
 
+    public static final String MSG_200_GET_HEARINGS = "Success (with content)";
+    public static final String MSG_400_GET_HEARINGS = "Invalid request";
+    public static final String MSG_200_DELETE_HEARING = "Success (with content)";
+    public static final String MSG_400_DELETE_HEARING = "Invalid request";
```

Comment:
As these are only referenced in the tests- can we move the static strings to there

Reply:
Done.